netutils/interface.py

```diff
@@ -118,16 +121,22 @@ def canonical_interface_name(interface, addl_name_map=None, verify=False):
     if isinstance(addl_name_map, dict):
         name_map.update(addl_name_map)
     # check in dict for mapping
-    if name_map.get(interface_type):
-        long_int = name_map.get(interface_type)
+    if interface_type in name_map:
```

Comment:
can we do `if interface_type in name_map.keys():`? I think the explicit speaks to the future developer looking at this seeing that it is a dictionary.

Reply:
Hm I personally don't like it - shouldn't we be able to mentally infer for the `_map` suffix that this variable is a dictionary? Also explicitly calling `.keys()` for `in` checks is not a pattern I have seen before as far as I can remember.

Reply:
Closing after slack conversation.